src/pages/Signup/Signup.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Signup.scss';
+
+class Signup extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      name: '',
+    };
+  }
+  handleInputValueChange = e => {
+    const { id, value } = e.target;
+    this.setState({
+      [id]: value,
+    });
+  };
+  checkValidation = e => {
+    e.preventDefault();
+    const { id, pw, name } = this.state;
+    if (id && pw && name) {
+      alert('success');
+    } else {
+      alert('Fail');
+    }
```

Comment:
```suggestion
    if (id && pw && name) {
      return alert('success');
    } 
      return alert('Fail');
    
```
- 삼항연산자를 사용해줄 수도 있겠지만, &&연산자로 인해 가독성이 조금 떨어진다면 이런식으로 수정 가능할 것 같습니다.
- else문은 최대한 지양해주는 것이 가독성에 좋습니다.

Reply:
넵